R/openxlsx2.R

```diff
@@ -19,34 +19,23 @@
 #' @importFrom zip zip
 #'
 #' @seealso
-#' \itemize{
-#'    \item{`vignette(package = "openxlsx2")`}
-#'    \item{[write_data()]}
-#'    \item{[write_datatable()]}
-#'    \item{[write_xlsx()]}
-#'    \item{[read_xlsx()]}
-#'   }
+#' * `browseVignettes("openxlsx2")`
+#' * [write_data()]
+#' * [write_datatable()]
+#' * [write_xlsx()]
+#' * [read_xlsx()]
+#' * <https://janmarvin.github.io/openxlsx2>
 #' for examples
 #'
 #' @details
-#' The openxlsx package uses global options to simplify formatting:
```

Comment:
Put options in their own .Rd file, see openxlsx2_options.Rd

Reply:
dito